client/header/activity-panel/panels/orders.js

```diff
@@ -28,12 +28,12 @@ import withSelect from 'wc-api/with-select';
 
 class OrdersPanel extends Component {
 	renderEmptyCard() {
-		const { hasNonActionableOrders } = this.props;
-		if ( hasNonActionableOrders ) {
+		const { hasActionableOrders } = this.props;
+		if ( hasActionableOrders ) {
```

Comment:
Aren't actionable orders the ones that need fulfillment?

`hasNonActionableOrders` is essentially "the store has received orders" - which is why its used to display "no **new** orders" vs. "you have no orders".

Reply:
`hasNonActionableOrders` isn't a prop so it was always defaulting to false. The PR now checks to see if there are any actionable orders in history for `hasActionableOrders` where the ones being shown in the panel are the new actionable orders.

Reply:
> `hasNonActionableOrders` isn't a prop so it was always defaulting to false.

I'm not sure I follow. `hasNonActionableOrders` is a computed prop here: https://github.com/woocommerce/woocommerce-admin/blob/8384d988ceed5868ee66b448edf91c0567ed7f34/client/header/activity-panel/panels/orders.js#L293. I do see the intended behavior of it in my testing.

Reply:
Possibly I was looking for the prop in the wrong place

<img width="998" alt="Screen Shot 2019-07-17 at 3 50 21 PM" src="https://user-images.githubusercontent.com/343847/61402988-af45ba80-a8aa-11e9-99d1-0ba6d9545bc4.png">

Reply:
Found it.
<img width="905" alt="Screen Shot 2019-07-17 at 3 57 43 PM" src="https://user-images.githubusercontent.com/343847/61403412-a86b7780-a8ab-11e9-8158-27444381e704.png">